modules/salt/pages/salt-terminology.adoc

```diff
@@ -29,46 +29,53 @@ Multiple Salt environments are not supported.
 Grain::
 Grains provide information about the hardware of a client.
 This includes the operating system, IP addresses, network interfaces, and memory.
-
++
 When you run a Salt command any modules and functions are run locally from the system being called.
-
++
 Salt modules are stored on clients and the {productname} Server within the [path]``/usr/lib/python*/site-packages/salt/`` directory.
-
++
 List all available grains with the [command]``grains.ls`` function:
++
 ----
 salt '*' grains.ls
 ----
-
++
 You can also use [command]``grains.items`` to list collected grain system data:
++
 ----
 salt '*' grains.items
 ----
-
++
 For more information on grains, see https://docs.saltstack.com/en/latest/topics/grains/.
 
 
+Highstate::
+This term is used when you apply all outstanding states to all targeted clients at the same time.
+The highstate must be applied regularly when doing broad changes to systems, including enabling and disabling formulas.
```

Comment:
```suggestion
The highstate must be applied when doing changes to systems, including enabling and disabling formulas.
```
I'd say it's not important whether a change is broad or small, and why appyling highstate if no state is outstanding?  But you can say something such as:
If you apply the highstate regularly, you are on the safe side: forgotten outstanding events will then be applied.